iothub/service/src/IotHubServiceClientOptions.cs

```diff
@@ -57,12 +57,12 @@ public class IotHubServiceClientOptions
         public HttpClient HttpClient { get; set; }
 
         /// <summary>
-        /// Whether to use web sockets or not.
+        /// The configured transport protocol.
         /// </summary>
         /// <remarks>
         /// Only used for AMQP. Can only be used for <see cref="MessagingClient"/> and <see cref="MessageFeedbackProcessorClient"/> and <see cref="FileUploadNotificationProcessorClient"/>.
         /// </remarks>
-        public bool UseWebSocketOnly { get; set; }
+        public TransportType Transport { get; set; } = TransportType.Tcp;
```

Comment:
In my view, the transport is HTTP or AMQP, and the protocol is TCP or web socket. Unless I got that wrong, I'd name this property `Protocol`.

Reply:
In the device client side, this enum was called ```IotHubClientTransportProtocol``` FYI

Reply:
Since it is the same concept here in the service client side, we should match the name used